### PR TITLE
add boolean expression documentation

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: en/user_guide/boolean_expression.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+* priority between the terms is specified by brackets,
+* parenthesis, beginning the expression, is followed by a space,
+* parenthesis, ending the expression, is preceded by a space,
+* parenthesis, nor beginning, nor ending is preceded and followed by a space,
+* parenthesis with no space will be considered as being in the attached keyword,
+* brackets can not be empty,
+* two operators are allowed, OR and AND,
+* operator is preceded and followed by a space,
+* expression can not beginning or ending with an operator,
+* two operators can not follow.
```
Comment (nicolasThal) [inline en/user_guide/boolean_expression.rst]: an operator cannot be followed by another operator ?

Comment (veolys-nanne) [inline en/user_guide/boolean_expression.rst]: No, you can't write AND AND, OR AND, AND OR, OR OR, @nicolasThal

Comment (veolys-nanne) [inline en/user_guide/boolean_expression.rst]: modification done

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**